bin/image-load

```diff
@@ -37,7 +40,7 @@ usage() {
 
 while :
 do
-  case $1 in
+  case ${1:-} in
```

Comment:
What was the reasoning for this change?

Reply:
We're sourcing _docker.sh which contains `set -u` which causes us to throw errors for unbound variables.  When we get to the end of the argument list `$1` is unbound so with `-u` set, we throw an error here.  Instead, I change it to default to empty so avoid the unbound variable error.

Reply:
Ah okay thanks for the explanation!